auth-web/src/components/auth/common/ModalDialog.vue

```diff
@@ -194,4 +194,10 @@ export default defineComponent({
   [title] {
     text-align: center;
   }
+  .short-name-lookup-dialog {
+    background: red;
```

Comment:
Probably a way to do this inline? Also why is there short name specific code in here? it should be generic

Reply:
Alternatively, rename the selector to `lookup-dialog`, I think that would work too